### PR TITLE
Modifed encryptor deployment to automount service account token

### PR DESCRIPTION
**Pull request by ncthbrt**
This resolves issue #33

### CODE REVIEW

File: charts/kamus/templates/deployment-encryptor.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "kamus.name" . }}
-      automountServiceAccountToken: false
+      automountServiceAccountToken: true
```
Comment (omerlh) [inline charts/kamus/templates/deployment-encryptor.yaml]: Maybe make it optional?
```suggestion
      automountServiceAccountToken: {{ .Values.automountServiceAccountToken}}
```

And set the default value to false?

Comment (ncthbrt) [reply charts/kamus/templates/deployment-encryptor.yaml]: That makes more sense. Is there anything else we'd need to add to the pr to register the value in the template?

Comment (omerlh) [reply charts/kamus/templates/deployment-encryptor.yaml]: No, just add it to the readme please

Comment (ncthbrt) [reply charts/kamus/templates/deployment-encryptor.yaml]: Updated the PR. Thanks

**Comment by ncthbrt** [comment]
Thanks @omerlh

**Comment by omerlh** [comment]
Thank you for your contribution! The new chart should be available any minute, once the build finished.

**Pull request merged**